site/src/examples/omezarr/omezarr-demo.tsx

```diff
@@ -43,6 +43,15 @@ const demoOptions: DemoOption[] = [
             url: 's3://allen-genetic-tools/tissuecyte/823818122/ome_zarr_conversion/823818122.zarr/',
         },
     },
+    {
+        value: 'opt5',
```

Comment:
We need to either switch "V3 Zarr Example Image (S3) (color channels: [R, G, B])" to "opt6" or move this one to the end of the list and name it "opt6". Right now the OME-Zarr V3 doesn't work because the key values are clashing

Reply:
sorry about that - fixed!